docs/source/maintenance.rst

```diff
@@ -0,0 +1,164 @@
+
+   * Every Sphinx documentation must contain an index.rst. Sphinx will generate index.html from this file, the index page of this documentation.
+   * This index.rst file must contain a toctree (.. toctree::) containing all "content" that is not included on the index page itself. As for now, 06.06.2023, this amounts to three sections, namely modules, examples and maintenance. Each of these sections has its own .rst file, modules.rst, examples.rst and maintenance.rst respectively. All .rst files which don't through toctrees have a path to this root toctree will not be part of the documentation.
+   * Further, you can find conf.py and as of now wip.gif. Both are not relevant for the sake of maintenance.
+   
+As of now, maintaining the documentation is equivalent to maintaining the developers documentation, i.e., the modules section.
+
+The modules section
+###################
+
+The modules section, for the sake of tidiness, consists of one modules.rst file in docs/source and the directory docs/source/modules. 
+
+   * The modules.rst file consists only of a toctree linking to all the .rst files in docs/source/modules. Thus, modules.rst must only be changed if we wish to document more or less than, as of now, psydac.api/ddm/feec/fem/linalg/mapping/polar/utilities.
+   * The docs/source/modules directory consists of, as of now, eight .rst files and eight folders, one of each for each of psydacs modules api/ddm/feec/fem/linalg/mapping/polar/utilities which are currently part of the documentation. If we also wanted to document psydac.cad for example, we'd have to add the respective structure here as well as add psydac.cad to the toctree of the modules.rst file as described above.
+   
+Individual modules
+##################
+
+Unless major changes take place, we only have to update existing files in the docs/source/modules directory, not add or remove any.
+As it was the case for the modules.rst file, each module .rst file (api.rst, ddm.rst, ...) consists only of a (hidden) toctree containing all of its submodules (api.basic, api.discretization, ...) as well as a visually more pleasing table which also highlights module docstrings.
+Hence, api.rst for example must only be updated if submodules are being added or deleted.
+
+More regularly though, pull requests will only add or delete classes and functions of submodules.
+Such changes are quickly applied to the documentation, as such changes must be addressed in one file only, see :ref:`here <simpleadd>`. The file in question is the .rst file of the corresponding submodule and it contains all classes and (public) functions of such a submodule.
+
+.. _simpleadd:
+
+Adding or deleting a class or a (public) function to a module
+-------------------------------------------------------------
+
+1) Adding a class
+
+With a coming PR, the class GMRES will be added to the psydac.linalg.solvers submodule.
```

Comment:
AFAIK `psydac.linalg` is a **subpackage** of `psydac`, and `psydac.linalg.solvers` is a **module**, i.e. it corresponds to an actual Python file.

Reply:
Ah okay, good to know! Will update throughout the documentation.

Reply:
3fa4e0272e920da64a042cbb87deb679ade437c5